### PR TITLE
Added error checking for lisa

### PR DESCRIPTION
**Pull request by mbivolan**

### CODE REVIEW

File: scripts/lis_hyperv_platform/lisa_run.ps1 [hunk 1]
```diff
@@ -42,12 +42,34 @@ function Edit-XmlTest {
...
+        }
+    }
+}
+
 function Main {
     pushd "$WorkDir"
     ($KeyName, $XmlName) = Get-Dependencies $KeyPath $XmlTest
     Edit-XmlTest $VMName $XmlName $KeyName 
     pushd ".\lis-test\WS2012R2\lisa\"
     .\lisa.ps1 run xml\$XmlName -dbg 3
```
Comment (ader1990) [inline scripts/lis_hyperv_platform/lisa_run.ps1]: isn't this code failing with an error -- don t you need to put a try/catch here?

File: scripts/lis_hyperv_platform/lisa_run.ps1 [hunk 2]
```diff
@@ -42,12 +42,34 @@ function Edit-XmlTest {
     popd
 }
 
+function Check-LisaExit {
+    param(
+        [int] $exitCode
```
Comment (ader1990) [inline scripts/lis_hyperv_platform/lisa_run.ps1]: $ExitCode

File: scripts/lis_hyperv_platform/lisa_run.ps1 [hunk 3]
```diff
@@ -42,12 +42,34 @@ function Edit-XmlTest {
...
+    )
+
+    $icaPath=".\TestResults\**\ica.log"
```
Comment (ader1990) [inline scripts/lis_hyperv_platform/lisa_run.ps1]: spaces after/before =

File: scripts/lis_hyperv_platform/lisa_run.ps1 [hunk 4]
```diff
@@ -42,12 +42,34 @@ function Edit-XmlTest {
...
+    if ($exitCode) {
```
Comment (ader1990) [inline scripts/lis_hyperv_platform/lisa_run.ps1]: to not indent too much code, as you do not have an else statement, you can just:
if (!ExitCode) { return }

continue with the unindented code
it makes the code more readable

File: scripts/lis_hyperv_platform/lisa_run.ps1 [hunk 5]
```diff
@@ -42,12 +42,34 @@ function Edit-XmlTest {
...
+        if (Test-Path $icaPath){
```
Comment (ader1990) [inline scripts/lis_hyperv_platform/lisa_run.ps1]: spaces

**Comment by chvalean** [comment]
Closing until further lisa stabilization into pipeline runs

**Pull request closed without merging**